tests/test.py

```diff
@@ -197,6 +198,27 @@ def test_corresponding_oauth2_implicit_flow_id_token_instance(self):
                          "scope='openid profile email')".replace('(', '\(').replace(')', '\)'))
 
 
+    def test_corresponding_oauth2_authorization_code_flow_instance(self):
```

Comment:
Can you add:

one test case with okta function for the one provided as well
2 tests cases (one with and without okta function) by specifying a non default value? (this one would have spotted the bug with the extra slash :) )